jmh-core/src/main/java/org/openjdk/jmh/runner/options/CommandLineOptions.java

```diff
@@ -131,7 +131,7 @@ public CommandLineOptions(String... argv) throws CommandLineOptionException {
                 .withRequiredArg().ofType(TimeValue.class).describedAs("time");
 
         OptionSpec<Integer> optThreads = parser.accepts("t", "Number of worker threads to run with. 'max' means the " +
-                "maximum number of hardware threads available on the machine, figured out by JMH itself. " +
+                "maximum number of hardware threads available on the machine, figured out by JMH itself. 'half_max' means 'max/2'" +
```

Comment:
I'd suggest we do `halfmax` instead of enum-style-but-not-really-enum `half_max`.